### PR TITLE
[s] Sets datum antag progress back about a month

### PR DESCRIPTION
**Pull request by Incoming5643**
Adds six new roundtypes, wow!

**Tensioner**
Random antags are made every so often!

**Everyone is the Traitor**
Everyone is the traitor!

**Everyone is the Changeling**
Everyone is the changeling!

**Everyone is the Traitor and also the Changeling**
Everyone is the traitor! Everyone is the changeling!

**Extended Events**
It's extended, but summon events is on full blast!

**Cleanbot**
Cleanbot!!!

---

By default can only be forced, if you want to give them proper probabilities add these lines to game_options.txt

PROBABILITY CLEANBOT 0
PROBABILITY EVERYCHANGELING 0
PROBABILITY EVERYTRAITOR 0
PROBABILITY EVERYTRAITORLING 0
PROBABILITY EXTENDEDEVENTS 0
PROBABILITY TENSIONER 0

### CODE REVIEW

**Comment by PKPenguin321** [comment]
c l e a n b o t

**Comment by Jazaen** [comment]
Praised b... WAAAAIT A MOMENT...

**Comment by Bawhoppen** [comment]
im going to add skub wars gamemode y/y

**Comment by Aloraydrel** [comment]
:washingrobot:

**Comment by Eaglendia** [comment]
:+1:

**Comment by Cheridan** [comment]
oh my god

File: code/game/gamemodes/memezone/everyoneisthechangeling.dm [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	reroll_friendly = 0
+	restricted_jobs = list()
+	protected_jobs = list()
+
+/datum/game_mode/changeling/everyone/announce()
+	world << "<B>The current game mode is - Everyone is the Changeling!</B>"
+	world << "<B>Due to a clerical error, literally every \"staff\" member on this station is actually a changeling. Life might be unexpectedly difficult!</B>"
+
+/datum/game_mode/changeling/everyone/pre_setup()
+	config.changeling_scaling_coeff = INFINITY + INFINITY
```
Comment (Jalleo) [inline code/game/gamemodes/memezone/everyoneisthechangeling.dm]: Heh infinity +infinity = infinity

**Comment by Bawhoppen** [comment]
cheri still refuses to acknowledges this is for april fools

**Comment by Shadowlight213** [comment]
No. This must be made real. Run sybil 24/7 tensioner

**Comment by AlexanderUlanH** [comment]
Summon events needs some new things for it to be a gamemode. In practice, it'll be extended with katanas vs magicarp, some bodyswapping, and nobody allowed to use internals.

**Comment by optimumtact** [comment]
They may as well put these on main as long as they're configurable

File: code/game/gamemodes/memezone/tensioner.dm [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+/datum/game_mode/tensioner/process()
+	doctor_incomings_mean_meme_machine++
+	if(doctor_incomings_mean_meme_machine > 500 * If_there_were_two_guys_on_the_moon_and_one_killed_the_other_with_a_rock_would_that_be_fucked_up_or_what)
+		if(Do_Something())
+			If_there_were_two_guys_on_the_moon_and_one_killed_the_other_with_a_rock_would_that_be_fucked_up_or_what++
+			doctor_incomings_mean_meme_machine = 0
+
+/datum/game_mode/tensioner/proc/Do_Something()
+	var/datum/admins/I = new
+	var/why_is_this_a_var = pick(I.makeTraitors(), I.makeChanglings(), I.makeRevs(), I.makeWizard(), I.makeCult(), I.makeNukeTeam(), I.makeAliens(), I.makeSpaceNinja(), I.makeDeathsquad(), I.makeGangsters(), I.makeOfficial(), I.makeAbductorTeam(), I.makeRevenant(), I.makeShadowling(), I.makeERPsquad())
```
Comment (Shadowlight213) [inline code/game/gamemodes/memezone/tensioner.dm]: makeERPsquad isn't a proc. this will runtime.

Comment (Incoming5643) [inline code/game/gamemodes/memezone/tensioner.dm]: it is on the april fools branch, thanks to #16311

Fucking JOKE SYNERGY

**Pull request merged**